insider_trades/helpers.py

```diff
@@ -1,74 +1,83 @@
                     isins.append("NA")
         transactions.set_isins(isins)
 
     def place_trades(self, buy: List[str], sell: List[str]):
-        orders = []
+        order_ids = []
 
-        expires_at = "p0d"
+        expires_at = 0
+        quantity = 1
 
         # place buy orders
         for isin in buy:
             side = "buy"
-            quantity = 1
-            order = self._lemon_api.place_order(
-                isin, expires_at, quantity, side
+            price = self.client.market_data.quotes.get_latest(isin=isin).results[0].b * quantity
+            if price < 50:
+                print(f"Order cannot be placed as total price, €{price}, is less than minimum order amount of €50.")
+                continue
+            order = self.client.trading.orders.create(
+                isin=isin, expires_at=expires_at, quantity=quantity, side=side
             )
-            orders.append(order)
+            order_ids.append(order.results.id)
             print(f"You are {side}ing {quantity} share(s) of instrument {isin}.")
 
-        portfolio = self._lemon_api.get_portfolio()
-        portfolio_isins = []
+        positions = self.client.trading.positions.get().results
+        positions_isins = []
 
-        for item in portfolio:
-            portfolio_isins.append(item.get("isin"))
+        for item in positions:
+            positions_isins.append(item.isin)
 
         # place sell orders
         for isin in sell:
-            if isin in portfolio_isins:
+            if isin in positions_isins:
                 side = "sell"
-                quantity = 1
-                order = self._lemon_api.place_order(
-                    isin, expires_at, quantity, side
+                price = self.client.market_data.quotes.get_latest(isin=isin).results[0].b / 10000. * quantity
+                if price < 50:
+                    print(f"Order cannot be placed as total price, €{price}, is less than minimum order amount of €50.")
+                    continue
+                order = self.client.trading.orders.create(
+                    isin=isin, expires_at=expires_at, quantity=quantity, side=side
                 )
-                orders.append(order)
+                order_ids.append(order.results.id)
                 print(f"You are {side}ing {quantity} share(s) of instrument {isin}.")
             else:
                 print(
                     f"You do not have sufficient holdings of instrument {isin} to place a sell order."
                 )
 
-        return orders
-
-    def activate_order(self, orders):
-        for order in orders:
-            self._lemon_api.activate_order(order["results"].get("id"))
-            print(f'Activated {order["results"].get("isin")}')
-        return orders
-
+        return order_ids
 
+    def activate_orders(self, order_ids):
+        for order_id in order_ids:
+            self.client.trading.orders.activate(order_id=order_id)
 
+    def get_opening_days(self):
```

Comment:
See following comments about the scheduler - if you decide to implement it, then this function is no longer needed.